service/keychain/cri/cri.go

```diff
@@ -71,10 +71,12 @@ func NewCRIKeychain(ctx context.Context, connectCRI func() (runtime.ImageService
 }
 
 type instrumentedService struct {
-	cri   runtime.ImageServiceClient
+	runtime_alpha.UnimplementedImageServiceServer
```

Comment:
This is required to match containerd's version of v1alpha api.

https://github.com/containerd/stargz-snapshotter/blob/4efc303653639d4e0b4faeb34b4c21dc8f1a15c6/service/keychain/crialpha/crialpha.go#L62